cmd/up.go

```diff
@@ -31,7 +31,7 @@ func UpCommand() cli.Command {
 	}
 }
 
-func ClusterUp(rkeConfig *v3.RancherKubernetesEngineConfig, dialerFactory hosts.DialerFactory) (string, string, string, string, error) {
+func ClusterUp(rkeConfig *v3.RancherKubernetesEngineConfig, dialerFactory, healthcheckFactory hosts.DialerFactory) (string, string, string, string, error) {
 	logrus.Infof("Building Kubernetes cluster")
 	var APIURL, caCrt, clientCert, clientKey string
 	kubeCluster, err := cluster.ParseCluster(rkeConfig, clusterFilePath, dialerFactory)
```

Comment:
may be it would make sense to set the healthcheckFactory on per cluster level instead of passing it as an argument to DeployClusterPlanes